api/v1alpha1/perconaservermysql_types.go

```diff
@@ -840,6 +840,16 @@ func (cr *PerconaServerMySQL) CheckNSetDefaults(ctx context.Context, serverVersi
 	return nil
 }
 
+func (cr *PerconaServerMySQL) FinalizerApplied(finalizer string) bool {
+	for _, f := range cr.GetFinalizers() {
+		if f == finalizer {
+			return true
+		}
+	}
+
+	return false
+}
+
```

Comment:
there's the same function in controllerutil: https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/controller/controllerutil#ContainsFinalizer

Reply:
Oh nice

Reply:
[e730d4f](https://github.com/percona/percona-server-mysql-operator/pull/726/commits/e730d4f7013d11e8cf931801667b496504ebfabe)